tests/posix/pty_uni.icn

```diff
@@ -32,26 +32,35 @@ $endif
    # test 2: prove you can actually read and write interactively, with
    # at least a line-at-a-time program, such as udb
    #
+   # the stand/pty_uni.std has been updated as there is now a different version
+   # for udb. Chnaged from 2.0 to 2.1
+   #
    fi := open("../../bin/udb", "prw") | stop("can't open udb mode prw")
    line := ""
    repeat {
-       if line ||:= ready(fi) then
+       if line ||:= ready(fi) then {
 	   write(image(line))
+        }
        if find("(udb)", line) then break
    }
    write("got initial udb prompt")
-   write(fi, "where"); flush(fi)
+   #
+   # remove flush() call as it is causing runtime memory violation
+   #
+   write(fi, "where"); #flush(fi)
```

Comment:
Maybe we should fix flush() instead of hiding it under the rug

Reply:
I wonder if  this is a problem with using flush with pipes!

Reply:
My c programming skills are not sufficient to investigate this matter at the moment. There is another matter with the pipes that I observed and that related to what the unicon program is seeing coming from the other end. When you senf a command to UDB, as you read the response, it appears to resend the same information many times with a terminating CR that would simply overwrite your display. Is this a feature of the pipes implementation or a bug?

An example in the data  transcript is the sequence:

" w\r         \r(udb) wh"
" w\r         \r(udb) wh\r"
The "\r" in effect returns the cursor to the same position each time and the following characters will overwrite what was there previously.   

When you run the UDB program, there does not appear to be this occurring. I think this needs investigation.